bench/Main.hs

```diff
@@ -236,17 +238,15 @@ runBench Bench {..} = handleAny (\e -> print e >> return (-1))
               loop (n -1)
 
     (t, res) <- duration $ loop samples
-
     exitServer
-    -- sleeep to give ghcide a chance to print the RTS stats
-    liftIO $ threadDelay 50000
-
     return $ if res then t else -1
   where
     cmd =
       unwords $
         [ "ghcide",
           "--lsp",
+          "--test",
```

Comment:
Why is this required?

Reply:
See https://github.com/digital-asset/ghcide/pull/624/files#diff-9f3c72df87c6e40a0ab15d33001d156cR739